ci/ciimage/arch/install.sh

```diff
@@ -18,7 +18,7 @@ pkgs=(
   # cuda
 )
 
-aur_pkgs=(scalapack wxwidgets-gtk2)
+aur_pkgs=(scalapack wxwidgets-gtk2 reflex-lexer)
```

Comment:
"RE/flex" https://www.genivia.com/reflex.html and "reflex" https://invisible-island.net/reflex/ are not the same project, but unfortunately AFAIK both of them install a `reflex` executable. I don't know which one is packaged in Arch with the `reflex-lexer` package name, but the commit message seems to refer to the first. AFAIK RE/flex's `reflex` is not a drop-in replacement for `flex`.

Reply:
That matches my understanding, yeah.